### PR TITLE
[Update] Using the Linode CLI

### PR DESCRIPTION
**Pull request by hzoppetti**
Added LKE and Object storage cli commands
Reordered the list to reflect the API page order

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 79a42c3fbb674700b8bc0f7c65a5485ae23f0cf4

https://deploy-preview-3137--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit faa934610fd67798f94ff713680e6248cc0b148c

https://deploy-preview-3137--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/platform/api/linode-cli/index.md [hunk 1]
```diff
@@ -249,6 +323,34 @@ Many other actions are available. Use `linode-cli linodes --help` for a complete
...
+
+        linode-cli object-storage keys-list
+
+1.  Update an Object Storage Key label:
+
+        linode-cli object-storage keys-update --keyId $key_id --label "my-new-object-storage-key"
+
+1.  Revoke an Object Storage Key:
+
+        linode-cli object-storage keys-delete --keyId $key_id
```
Comment (andystevensname) [inline docs/platform/api/linode-cli/index.md]: This might not accept the --keyId argument. Might just needs keys-delete and the key ID.

Comment (hzoppetti) [reply docs/platform/api/linode-cli/index.md]: correct - removed

**Pull request merged**